src/EnsembleKalmanInversions.jl

```diff
@@ -470,40 +469,42 @@ Iterate the ensemble Kalman inversion problem `eki` forward by `iterations`.
 function iterate!(eki::EnsembleKalmanInversion; iterations = 1)
 
     for _ in ProgressBar(1:iterations)
-    
+
         θ = get_u_final(eki.ensemble_kalman_process) # (N_params, ensemble_size) array
         G = eki.inverting_forward_map(θ) # (len(G), ensemble_size)
-    
+
+        @show size(G)
```

Comment:
Is this intentional or was there for debugging purposes?

Reply:
Resolved!